### PR TITLE
Upgrade existing Compute networks models & requests

### PR DESCRIPTION
**Pull request by emilymye**
I'll probably hold off on adding new requests until we have everything else updated.

### CODE REVIEW

File: lib/fog/compute/google/models/network.rb [hunk 1]
```diff
@@ -8,19 +8,24 @@ class Google
...
-        attribute :kind
-        attribute :id
-        attribute :ipv4_range, :aliases => "IPv4Range"
-        attribute :creation_timestamp, :aliases => "creationTimestamp"
+        attribute :auto_create_subnetworks, aliases => "autoCreateSubnetworks"
+        attribute :creation_timestamp, aliases => "creationTimestamp"
         attribute :description
-        attribute :gateway_ipv4, :aliases => "gatewayIPv4"
-        attribute :self_link, :aliases => "selfLink"
+        attribute :gateway_i_pv4, aliases => "gatewayIPv4"
```
Comment (icco) [inline lib/fog/compute/google/models/network.rb]: Anyway we can keep `ip` together? Or is this Rubocop complaining?

Comment (emilymye) [reply lib/fog/compute/google/models/network.rb]: I changed it to match the argument in the Google Client API. I'm not sure if we want it to vary - i could add it as an alias? Mostly we would have to do some conversion during insert.

Comment (icco) [reply lib/fog/compute/google/models/network.rb]: Ah ok. Let us do an alias. If it's an alias, I think the conversion should happen automatically.

Comment (emilymye) [reply lib/fog/compute/google/models/network.rb]: done!

**Pull request merged**